src/components/Navbar/Navbar.vue

```diff
@@ -0,0 +1,1259 @@
+<script setup>
+const { locales, setLocale } = useI18n();
+
+const openDropdown = ref(false);
+const loggedIn = ref(false);
+function handleDropdownButton(dropdown) {
+	if (!openDropdown.value) {
+		openDropdown.value = dropdown;
+	} else if (openDropdown.value === dropdown) {
+		openDropdown.value = false;
+	} else {
+		openDropdown.value = dropdown;
+	}
+}
+</script>
+
+<template>
+  <header class="transparent">
+    <div class="left-section">
+      <button
+        id="mobile-button"
+        class="dropdown-button"
```

Comment:
Let's remove the `id` attribute from the buttons, there's no use for it now (plus it can cause problems later, browsers don't like multiple ids).

There are a few more ids in this file, I wont add a review comment for all of them cuz its a bit spammy.

You can keep the `clippath` ids as is. It does cause problems but those existed in the original codebase so I don't think its a problem. (For now)